tests/pages/SuppressionJeune.page.test.tsx

```diff
@@ -0,0 +1,224 @@
+          hasSession: true,
+          session: { accessToken: 'accessToken' },
+        })
+
+        jeunesService = mockedJeunesService({
+          supprimerJeune: jest.fn(() => Promise.resolve()),
+        })
+        ;(withDependance as jest.Mock).mockReturnValue(jeunesService)
+      })
+
+      describe("quand le jeune s'est déjà connecté", () => {
+        let jeune: Jeune
+        let actual: any
+        beforeEach(async () => {
+          // Given
+          jeune = unJeune({ isActivated: true })
+          ;(jeunesService.getJeuneDetails as jest.Mock).mockResolvedValue(jeune)
+
+          // When
+          actual = await getServerSideProps({
+            query: { jeune_id: jeune.id },
+          } as unknown as GetServerSidePropsContext)
+        })
+
+        it('récupère le jeune', () => {
+          // Then
+          expect(jeunesService.getJeuneDetails).toHaveBeenCalledWith(
+            jeune.id,
+            'accessToken'
+          )
+        })
+
+        it('renvoie sur la fiche du jeune', () => {
+          // Then
+          expect(actual).toEqual({
+            redirect: {
+              destination: `/mes-jeunes/${jeune.id}`,
+              permanent: true,
+            },
+          })
+        })
+      })
+
+      describe("quand le jeune ne s'est jamais connecté", () => {
+        it('fournit le jeune à la page', async () => {
+          // Given
+          const jeune = unJeune({ isActivated: false })
+          ;(jeunesService.getJeuneDetails as jest.Mock).mockResolvedValue(jeune)
+
+          // When
+          const actual = await getServerSideProps({
+            query: { jeune_id: jeune.id },
+          } as unknown as GetServerSidePropsContext)
+
+          // Then
+          expect(actual).toEqual({ props: { jeune } })
+        })
+      })
+    })
+  })
+
+  describe('client side', () => {
+    let jeune: Jeune
+    let jeunesService: JeunesService
+    let push: jest.Mock
+    beforeEach(() => {
+      jeune = unJeune({
+        firstName: 'Nadia',
+        lastName: 'Sanfamiye',
+        email: 'nadia.sanfamiye@email.fr',
+      })
+      jeunesService = mockedJeunesService()
+      push = jest.fn(() => Promise.resolve())
+      ;(useRouter as jest.Mock).mockReturnValue({ push })
+
+      renderWithSession(
+        <DIProvider dependances={{ jeunesService }}>
+          <SuppressionJeune jeune={jeune} />
+        </DIProvider>
+      )
+    })
+
+    it('permet de revenir à la page précédente', () => {
+      // Then
+      const link = screen.getByText('Détails Nadia Sanfamiye')
+      expect(link).toBeInTheDocument()
+      expect(link.closest('a')).toHaveAttribute(
+        'href',
+        `/mes-jeunes/${jeune.id}`
+      )
+    })
+
+    it('titre la page', () => {
+      // Then
+      expect(
+        screen.getByText('Suppression du compte de Nadia Sanfamiye')
+      ).toBeInTheDocument()
+    })
+
+    it('rappel les information du jeune', () => {
```

Comment:
```suggestion
    it('rappel des information du jeune', () => {
```
sinon rappel # rappelle